### ISSUE TITLE
Move to the appimage github project

### ISSUE THREAD

**Issue by azubieta** [azubieta/appimage-info#10]
@probonopd I would like you to review the project in order to move it into the AppImage org.
@TheAssassin  already suggested to rename the lib to libappimageinfo and the cli command to libappimageinfotool.
I actually don't think that the "tool" sufix is required.

**Comment by TheAssassin** [comment]
`s/libappimageinfotool/appimageinfotool/`

**Comment by probonopd** [comment]
As per our naming policy
* GUI Applications get `AppImageCamelCases`
* Command line tools get `appimagefoobar[tool]` names

This mirrors the Mac policy.

But I am not sure this should be a part of the AppImage project at the moment. The AppImage project does not aim to collect any and every tool that is somehow related to AppImage. We are totally fine with projects like Nitrux, KDE,... to maintain independent code bases that do things with AppImages, without wanting this code in the AppImage project. This is especially true as long as no other code in the AppImage project depends on this.

**Comment by azubieta** [comment]
Actually, we have plans of making AppImageLauncher use the libappimageinfo code in order to show a bit more of "info" about the AppImage being opened. And this tool aims to be used not only at Nitrux or KDE but also on GNome, XFCE or any other desktop. Also, it could be used in software centers or file managers.

**Comment by probonopd** [comment]
Ah, in this case, couldn't we merge it into libappimage?

**Comment by TheAssassin** [comment]
No. That'd add _way_ too many dependencies to libappimage. We should keep it a separate project.
